### ISSUE TITLE
Crush when implementing interface

### ISSUE THREAD

**Issue by linonetwo** [Mayank1791989/gql#56]
when there is an interface, and we write an empty type implementing it, it crashes.

```graphql
interface Object {
  ID: ID!
  properties: [Property]!
}

type DocumentObject implements Object {

}
```

Vscode said it crashes too many times so won't start again.

**Comment by Mayank1791989** [comment]
I think this bug is already fixed in v2.3.1 (commit d36d075). Make sure you are using the lastest version.

**Comment by linonetwo** [comment]
thanks!
